pkg/bus/meseeks.go

```diff
@@ -699,10 +718,29 @@ func (b *Bus) openSeeks(ctx context.Context, receiverID string, tourneyID string
 	if len(sgs) == 0 {
 		return nil, nil
 	}
+
+	receiver, err := b.userStore.GetByUUID(ctx, receiverID)
```

Comment:
does this work when not logged in?
see #1138

Reply:
don't you have to be logged in to play? I thought we didn't allow anonymous play

Reply:
- for lobby, from `initRealmInfo`, it seems [`sendLobbyContext`](https://github.com/domino14/liwords/blob/891b10ba4e7a4337225ddec6121256734d11ba5b/pkg/bus/bus.go#L1024) sends `openSeeks` and then `activeGames` even if `userIsAnon`
- `openSeeks` fails (an ephemeral random userid is assigned for each anonymous tab, doing `GetByUuid` on these returns "user not found"), so `activeGames` isn't sent out, possibly causing #1138

(frontend doesn't currently show `openSeeks` result, but seems to maintain it internally anyway)

Reply:
(you do have to be logged in to play, but not just to see list of active games, watch them, and possibly see list of seeks)

[if there's no sought games](https://github.com/domino14/liwords/blob/891b10ba4e7a4337225ddec6121256734d11ba5b/pkg/bus/meseeks.go#L718) `GetByUUID` is skipped and the anonymous user can see list of active games. (hence it's hard to see this happening locally)

Reply:
Thank you for the explanation, yes,  I didn't see this issue in my local. I'll try to fix it